app/src/main/java/fr/free/nrw/commons/quiz/QuizQuestion.java

```diff
@@ -0,0 +1,62 @@
+package fr.free.nrw.commons.quiz;
+
+import android.net.Uri;
+
+/**
+ * class contains information about all the quiz questions
+ */
+public class QuizQuestion {
+    private int questionNumber;
+    private String question;
+    private boolean answer;
+    private String url;
+    private String answerMessage;
+
+    QuizQuestion( int questionNumber, String question, String url, boolean answer, String answerMessage){
```

Comment:
Please ensure that you use consistent whitespaces. We don't use whitespaces before the first parameter of a method, and there is usually a whitespace between the `)` and the `{`. Please check the rest of your code as well, as there are a few more instances of this. Thanks!